src/main/java/core/basesyntax/model/StorageTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.model;
+
+public class StorageTransaction {
+    private TypeActivity typeActivity;
+    private String fruit;
+    private int quantity;
+
+    public StorageTransaction(TypeActivity typeActivities, String fruit, int quantity) {
+        this.typeActivity = typeActivities;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public TypeActivity getTypeActivity() {
+        return typeActivity;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public enum TypeActivity {
```

Comment:
Better rename to Operation. It's more understandable 
![image](https://user-images.githubusercontent.com/32339836/224640961-71ccf654-11d4-4ba4-a52d-2cacee3b1a42.png)